Makefile

```diff
@@ -52,6 +52,12 @@ format:
 check: format-check linter-check
 	@true
 
+# This needs the following packages on Ubuntu: openssl softhsm2 opensc libengine-pkcs11-openssl
+test-pki:
```

Comment:
Maybe separate targets for each HSM type?
Also, I suppose, we can run the `-tags testhsm` type in CI?

Reply:
We do run it in CI.

If someone wants to run it separately - they can do `go test` themselves.
I prefer to keep the Makefile concise enough.